tutorials/boundary_conditions.ipynb

```diff
@@ -14,7 +14,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #84.    pp.run_time_dependent_model(model=model, params=solver_params)
</pre><p>Again, solver_params is redundant.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pmgbergen/porepy/pull/1218/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>